### DISCUSSION TITLE
Pdf export ** Heading followed by 1. (list) causes page break?

### DISCUSSION THREAD

**Discussion by mistyn8** [GitbookIO/community#273] [5. Export]
Seem to get unexpected page breaks when exporting to PDF. Any suggestions or just a bug? 

```
You can sort pages in two ways:

## Option 1

1. Go to **Content**.
2. Navigate to the parent node whose child nodes you wish to sort.
3.  Right-click the parent node and select **Sort**. Alternatively, click **•••** next to the parent node and select **Sort**.
```

![image](https://github.com/GitbookIO/community/assets/12418214/53574bdb-cc78-4cd5-a62c-a602db658489)

**Comment by aikoii** [comment]
Hey, thanks for bringing this up! We're definitely aware of the most issues with PDF export and we _might_ tackle them when we revamp the feature - but I can't give you a specific timeline for that. Just want to be transparent with you - it might not happen anytime soon 🙏. Our focus with GitBook is mainly on the web experience.

Related to #90
